src/pages/costModels/createCostModelWizard/markup.tsx

```diff
@@ -1,79 +1,115 @@
 import {
   FormGroup,
+  Grid,
+  GridItem,
   InputGroup,
   InputGroupText,
+  List,
+  ListItem,
+  Radio,
   Stack,
   StackItem,
   Text,
   TextContent,
   TextInput,
-  TextVariants,
   Title,
 } from '@patternfly/react-core';
 import { Form } from 'components/forms/form';
 import React from 'react';
 import { WithTranslation, withTranslation } from 'react-i18next';
 
+import { styles } from '../costModel/costCalc.styles';
 import { CostModelContext } from './context';
 
-interface MarkupValidationState {
-  isValid: boolean;
-}
-
-class Markup extends React.Component<WithTranslation, MarkupValidationState> {
-  public state = {
-    isValid: true,
-  };
-
+class Markup extends React.Component<WithTranslation> {
   public render() {
     const { t } = this.props;
-    const { isValid } = this.state;
 
     return (
       <CostModelContext.Consumer>
-        {({ onMarkupChange, markup }) => {
+        {({ handleSignChange, handleOnKeyDown, handleMarkupDiscountChange, markupValidator, markup, isDiscount }) => {
           return (
             <Stack hasGutter>
               <StackItem>
                 <Title headingLevel="h2" size="xl">
-                  {t('cost_models_wizard.markup.title')}
+                  {t('cost_calculations')}
                 </Title>
```

Comment:
I believe all `<Title>` components are expected to be wrapped by `<TextContent>` to have the proper spacing. They're used in a lot of places, so I'll let you search for them.

http://patternfly-react.surge.sh/components/text/

Reply:
Doing a search in our code base and I do not see any <Title> tags wrapped in  <TextContent>. If we think this is a pattern we should be following i can open another issues to track a sweep of the code.

Reply:
Sorry, I was speaking about the `Text` component. You replaced `Title` with `Text` and got rid of the `h6`

Reply:
Added headingLevel back to the none description text fields